server/src/routes/api/v1/attractionsReviewsRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+
+import { Review } from "../../../models/index.js";
+
+const attractionsReviewsRouter= new express.Router()
+
+attractionsReviewsRouter.get("/", async (req, res) => {
+    
+    try {
```

Comment:
we can remove the empty space on line 8

actually, this router isn't used at all and should be removed!